internal/utils/constants/db.go

```diff
@@ -4,5 +4,13 @@ const (
 	// TODO: Fall back to these values if the environment variables are not set
 	DB_DEFAULT_TIMEZONE = "Asia/Singapore"
 	DB_DEFAULT_PORT     = 5432
-	DB_DEFAULT_NAME     = "sa-stories"
+	DB_DEFAULT_NAME     = "sa_stories"
 )
+
+func GetOrDefault[T comparable](val T, deflt T) T {
```

Comment:
Nice abstraction! But I don't think there's any "getting" involved here – would it be better if the function is simply named `OrDefault`?

_(Of course, it won't be used right now due to my above comment)_